cluster/manifests/postgres-operator/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-operator
+data:
+  service_account_name: operator
```

Comment:
Lets create it then and have an example with it. should not be the same as system imho